typescript/src/scss/main.scss

```diff
@@ -6,11 +14,58 @@
     margin: 20px 0;
 
     td, th {
-      text-align: right;
+      text-align: center;
       padding: 2px 5px;
-      vertical-align: top;
       border: 0;
+      justify-content: center;
+    }
+    // TODO: remove .bottom tags from calculator.html and change .bottom to tfoot when the js issue is fixed.
+    thead, .bottom {
+      background-color: $light_blue;
+      padding: 1.2%
+    }
+    td, .result {
+      text-align: right;
+      padding-right: 20px;
+    }
+    tr:nth-child(even) {
+      background-color: $row_gray;
+    }
+      // TODO:  Change the tfoot to tbody when thejs issue is fixed to fix the overall shading
+    tfoot>tr>:nth-child(2), tfoot>tr>:nth-child(3),tfoot>tr>:nth-child(4),tfoot>tr>:nth-child(5) {
```

Comment:
Could you use a class for this rather than `:nth-child`? Hopefully you can see how to add the class to the dynamic rows in typescript but give me a shout if not. (It's less likely to break that way, e.g. if we added columns at any point.)